src/test/java/com/jcabi/log/ObjectDecorTest.java

```diff
@@ -77,13 +71,13 @@ public ObjectDecorTest(final Object object, final String text,
                 {null, "NULL", 0, 0, 0},
                 {new SecretDecor("x"), "{secret: \"x\"}", 0, 0, 0},
                 {new Foo(1, "one"), "{num: \"1\", name: \"one\"}", 0, 0, 0},
-            //  @checkstyle MethodBodyComments (6 lines)
-            //  @checkstyle LineLength (3 lines)
-            //  {
-            //      new Object[]{new Foo(0, "zero"), new Foo(2, "two")},
-            //      "[{num: \"0\", name: \"zero\"}, \"{num: \"2\", name: \"two\"}]",
-            //      0, 0, 0
-            //  },
+                //  @checkstyle MethodBodyComments (6 lines)
+                //  @checkstyle LineLength (3 lines)
+                {
```

Comment:
one more test example is missing - array, which contains another array. This case described in your task description